mpl_sphinx_theme/__init__.py

```diff
@@ -3,6 +3,32 @@
 from pathlib import Path
 
 
+def set_config_defaults(app):
+
+    try:
+        theme = app.builder.theme_options
+    except AttributeError:
+        theme = None
+    if not theme:
+        theme = {}
+
+    # Default logo
+    logo = {} if "logo" not in theme else theme["logo"]
+    if not logo:
+        logo = {}
```

Comment:
```suggestion
    logo = theme.get("logo")
    if not logo:
        logo = {}
```

Reply:
```suggestion
    theme.get("logo", {})
```

Any reason that this one-liner doesn't work here?

Reply:
Well, @jklymak just copied to code from jupyterbooks ;-). I'm not sure what the point is of the extra checks are...

Reply:
Yeah, I am not super well versed in how trusted/structured the input data is here, so not  offended by the extra checks for Falsy entries in the dict, though I do think that `.get` is preferable to the ternary operator as more readable/shorter.

Reply:
Sure, let's just go with the shorter since this is not meant to be a general-use theme.  We can always fix it if needed...